diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/KdocMethods.kt

```diff
@@ -77,7 +77,12 @@ class KdocMethods(configRules: List<RulesConfig>) : DiktatRule(
      * @param node
      */
     override fun logic(node: ASTNode) {
-        if (node.elementType == FUN && node.getFirstChildWithType(MODIFIER_LIST).isAccessibleOutside() && !node.isOverridden()) {
+        val isSuccessModifier : Boolean by lazy {
```

Comment:
Why are you making it lazy? It should be recalculated for every node, and since it's a local property, it will be recalculated anyway.

Reply:
OK, got it, it's calculated only if the first condition in `if` is true

Reply:
Because we need to recalculate this property only for `FUN` nodes.